server/app.js

```diff
@@ -209,8 +213,10 @@ const onSocketConnect = (io, socket, netid) => {
     console.log("Sending profile: " + JSON.stringify(profile))
     socket.emit("profile", profile)
     socket.emit("user_id", user_id)
-
+    clientCount++;
```

Comment:
Given the way this is implemented, this *should* be the same as `connections.size`, as every new connection means a new entry is added to the mapping (and each disconnect removes an entry). If that gives the same behavior, it might be cleaner to use the size property over keeping a separate counter.

Reply:
Changed; it does indeed have the same behavior